src/webviews/src/actions/oauthCallback.ts

```diff
@@ -59,7 +59,7 @@ export const processOauthCallback = ({ status, message = '', error }: CloudAuthR
     //   return
     // }
 
-    // dispatch(fetchPlans())
+    fetchPlans()
```

Comment:
This is the callback for oauth messages. So there are few domains in this - `SSO`, `Cloud stuff`, etc... IMO it would be more easier to track what's happening if:
```suggestion
    fetchSubscriptionsPlans()
```
or
```suggestion
    fetchCloudSubscriptionsPlans()
```
or something more descriptive. I know that we don't have other `plans`, so technically its OK.